carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml

```diff
@@ -55,3 +55,15 @@ services:
       - /opt/carma/.ros:/opt/carma/.ros
     command: bash -c 'wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/drivers.launch drivers:=dsrc_driver'
 
+  torc-pinpoint-driver:
+    image: usdotfhwastoldev/carma-torc-pinpoint-driver:develop
+    container_name: carma-torc-pinpoint-driver
+    network_mode: host
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.0.51
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/opt/carma/.ros
+    command: bash -c 'wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/drivers.launch drivers:=pinpoint_driver'
```

Comment:
The drivers.launch file needs to be updated to support this argument and actually launch the pinpoint driver. 
This the launch file we would use, the specific ip address will need to be passed in
https://github.com/usdot-fhwa-stol/carma-torc-pinpoint-driver/blob/develop/pinpoint/launch/pinpoint.launch

Reply:
c5f04e9